app/pages/project/project-page.cjsx

```diff
@@ -158,14 +159,30 @@ ProjectPage = React.createClass
           <Translate content="project.nav.collections" />
         </Link>
 
+        {if @props.organization
+          <Link to="/organizations/#{this.props.organization.slug}" className="tabbed-content-tab">
+            {this.props.organization.display_name}
+            {' '}
+            <i className="fa fa-external-link fa-fw" />
+          </Link>}
+
         {rearrangedLinks.map ({label, url}, i) =>
-          unless !!label
+          if !!label
```

Comment:
Could this be done without adding another `if-else` block to the already messy navbar logic? At least extract it to its own function? FYI, I'm preparing a PR that extracts the navbar to its own component and refactors it.

Reply:
I think so, what you're doing certainly sounds preferable. I didn't refactor the general links logic with this PR, so I'm sure what you're doing will be an improvement. 

I'll refactor this PR to add the link in the sub-nav to a project's organization, if applicable, to work with your changes.

Reply:
I'm already struggling with reconstructing the original logic. So I'll leave adding the external link icon to you, if that's ok.

Reply:
sounds good, I'll add the external link icon with this PR